azure-stack/user/azure-stack-vm-sizes.md

```diff
@@ -23,7 +23,8 @@ Disk IOPS (Input/Output Operations Per Second) on Azure Stack Hub is a function
 
 VM vCPU depends on the number of cores per node. For example, systems with cores or logical processor of less than 64 will not support VM size Standard_F64s_v2.
 
-## VM general purpose
+
+## VM General Purpose
 
```

Comment:
@NeilBird Please see the comment from PR reviewers -- the original heading was correct, please don't change it. Thank you!